.github/workflows/python-build-and-test.yml

```diff
@@ -1,21 +1,57 @@
-name: Python unit tests
+name: Nightly-check-db
```

Comment:
you should have separate workflow instead of overwriting current one

nit: also, naming should be consistent as `nightly-check-db`